src/expandableCalendar/week.tsx

```diff
@@ -36,14 +50,16 @@ const Week = (props: WeekProps) => {
         return <View key={id} style={style.current.emptyDayContainer}/>;
       }
     }
+    const dayString = toMarkingFormat(day);
 
     return (
       <View style={style.current.dayContainer} key={id}>
         <Day
           {...dayProps}
-          date={toMarkingFormat(day)}
+          testID={`${testID}_week_${dayString}`}
```

Comment:
The testID pattern used so far is `${testID}**.**week_${dayString}`. Also, why is it named "week" and not "week_day"?

Reply:
This is actually your change, not mine. 
I'll fix it

Reply:
Hmm now the testIDs are like this 
for calendarList
`expandableCalendar.calendarList.item_2022-08.day_2022-09-01`
for weekCalendar
`expandableCalendar.weekCalendar.day_2022-09-10`

Reply:
Make sense, except the calendarList item's date doesn't match the day date...

Reply:
That's on purpose, the calendar list item represents a year and a month, not a day. 
The day tends to change because it's based on the `current` value, it can be the first day sometimes or a the mid day of the month. 
When a user wants to fetch a specific calendar item, they should fetch it by year+month

Reply:
I meant this: `expandableCalendar.calendarList.item_**2022-08**.day_**2022-09**-01`

Reply:
Is it possible it's because we render days from nearby months as disabled days?

Reply:
Do you mean because of the extra days? mmm... if one of these days is selected the calendar is moved to that day's month, and if it is the 'current' it should show that day's month... I think haha